opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -74,25 +75,28 @@ public Span setTag(String key, Number value) {
 
   @Override
   public Span log(Map<String, ?> fields) {
-    // TODO
+    // TODO - verify 'null' for 'event' is valid.
+    span.addEvent(null, convertToAttributes(fields));
     return this;
   }
 
   @Override
   public Span log(long timestampMicroseconds, Map<String, ?> fields) {
-    // TODO
+    // TODO - use timestampMicroseconds
```

Comment:
in Open Census the idea was that you cannot provide your own timestamps in any tracking API. Only when constructing `SpanData`.

Reply:
Well, we somehow need to support this (same case as in `Span.end()`)

Reply:
we decided that this will not be supported